src/pybind/mgr/ssh/module.py

```diff
@@ -115,6 +97,23 @@ def __init__(self):
     def __getattr__(self, name):
         return getattr(self.conn, name)
 
+
+def log_exceptions(f):
```

Comment:
@tserong  do we need this also in the deepsea orchestrator?

Reply:
I have no idea :-) but as it seems to only add something in the python2 case, I'd be inclined to leave it out as we're targeting python3 for the future anyway